pkg/kv/dist_sender.go

```diff
@@ -59,6 +60,36 @@ const (
 	defaultSenderConcurrency = 500
 )
 
+var (
+	// DistSender Metrics
+	metaTotalSends         = metric.Metadata{Name: "distsender.totalSends"}
```

Comment:
@mberhault Do we have a preferred naming convention for metrics? For example, the names here use camel-case. In `storage/metrics.go` metric names appear to be lower case separated by hyphens.